client/src/app/components/AboutUs/about-us.css

```diff
@@ -0,0 +1,80 @@
+.about-us {
+  text-align: center;
+}
+
+.aboutBackground {
+  position: absolute;
+  left: 0;
+  z-index: 1;
+  background-image: url("https://images.pexels.com/photos/1068523/pexels-photo-1068523.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
+  background-size: cover;
+  background-position: center;
+  filter: brightness(45%);
+  width: 100%;
+  height: 400px;
+  transform: translate(100);
+}
+
+.content {
+  position: absolute;
+  left: 0;
+  right: 0;
+  z-index: 3;
```

Comment:
Whenever possible, using z-index should be avoided IMO. They can become difficult to manage, and prone to bugs difficult to detect. Usually the layer can be constructed in a way that doesn't make use of them.